src/runtime/bootstrap-lazy.ts

```diff
@@ -152,8 +155,8 @@ export const bootstrapLazy = (lazyBundles: d.LazyBundlesRuntimeData, options: d.
         if (BUILD.appendChildSlotFix) {
           patchSlotAppendChild(HostElement.prototype);
         }
-        if (BUILD.scopedSlotTextContentFix) {
-          patchTextContent(HostElement.prototype, cmpMeta);
+        if (BUILD.scopedSlotTextContentFix && BUILD.scoped && cmpMeta.$flags$ & CMP_FLAGS.scopedCssEncapsulation) {
```

Comment:
This check is the same that was in the `patchTextContent()` method. Just moved it to this level so we don't have to make changes in the future when we remove these `extras` flags.